.github/workflows/e2e-test.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: "Check Sample App Starts Up"
         working-directory: ./sample-code
         run: |
-          pnpm run local &
+          pnpm local &
```

Comment:
I am pretty sure I read somwhere in the pnpm docs, that it is recommended to use `run` in CI. I honestly don't care much as I don't see the risk of omitting `run`. So go for it, just keep this in the back of your mind if issues do occur.

Reply:
Hmm I see, I wanted to get rid of the inconsistent use of `run`, if we see issues occurring, I would vote for adding it to every pnpm command.